### PR TITLE
Truncate page numbering on Review pages (T186497)

### PR DESCRIPTION
**Pull request by uyscuti-wiki**
Limited the number of pages displayed on the review page.

### CODE REVIEW

**Comment by uyscuti-wiki** [comment]
Once you get to page 10, the format will be - Previous page, 1, 2, 3, ..., 9, 10 (focused), 11, Next page, Last page

**Comment by Samwalton9** [comment]
When on a page lower than 10, it's not obvious that there are more than 10 pages. Maybe there could be a ... and then the last few page numbers before 'Next page'? That should probably always be the case actually, so it's more like 1 2 3 ... 12 **13** 14 ... 26 27 28 (assuming there are 28 pages). That way the user always has context of the maximum number of pages.

**Comment by uyscuti-wiki** [comment]
A user will now be able to skip 9-11 pages at a time. Including the last three pages is possible, but I'd go with only one (to reduce clutter). Being able to skip pages is more practical, don't you think?

**Comment by uyscuti-wiki** [comment]
should now display '...' at the end signifying there's more

File: TWLight/applications/templates/applications/application_list.html [hunk 1]
```diff
@@ -109,17 +109,102 @@ <h1>{{ title }}</h1>
...
+			{% endfor %}
+		  {% elif object_list.number >= 10 %}		
+			{% for _ in range|slice:"3" %}
+				<li>
+					<a href="?page={{ forloop.counter }}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+						{{ forloop.counter }}
+					</a>
+				</li>
+            {% endfor %}
+			<li>
```
Comment (jsnshrmn) [inline TWLight/applications/templates/applications/application_list.html]: lines 131 .. 135 could be one line

File: TWLight/applications/templates/applications/application_list.html [hunk 2]
```diff
@@ -109,17 +109,102 @@ <h1>{{ title }}</h1>
...
-                {{ forloop.counter }}
+				{% endifequal %}>
+					<a href="?page={{ forloop.counter }}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+						{{ forloop.counter }}
+					</a>
+				</li>
+			{% endfor %}
+		  {% elif object_list.number >= 10 %}		
+			{% for _ in range|slice:"3" %}
+				<li>
```
Comment (jsnshrmn) [inline TWLight/applications/templates/applications/application_list.html]: lines 125 126 could be one line

File: TWLight/applications/templates/applications/application_list.html [hunk 3]
```diff
@@ -109,17 +109,102 @@ <h1>{{ title }}</h1>
...
+					<a href="?page={{ forloop.counter }}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+						{{ forloop.counter }}
+					</a>
```
Comment (jsnshrmn) [inline TWLight/applications/templates/applications/application_list.html]: lines 128 129 could be one line

File: TWLight/applications/templates/applications/application_list.html [hunk 4]
```diff
@@ -109,17 +109,102 @@ <h1>{{ title }}</h1>
...
+            </li>
+            <li class="active">
+              <a href="?page={{ object_list.number }}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+				{{ object_list.number }}
               </a>
             </li>
-          {% endfor %}
-        {% endwith %}
+			{% if object_list.has_next %}
+				<li>
```
Comment (jsnshrmn) [inline TWLight/applications/templates/applications/application_list.html]: lines 147 148 could be one line

File: TWLight/applications/templates/applications/application_list.html [hunk 5]
```diff
@@ -109,17 +109,102 @@ <h1>{{ title }}</h1>
...
+					<a href="?page={{ object_list.next_page_number }}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+						{{ object_list.next_page_number }}
+					</a>
```
Comment (jsnshrmn) [inline TWLight/applications/templates/applications/application_list.html]: lines 150 151 could be one line

**Comment by jsnshrmn** [comment]
Looks great!

**Comment by uyscuti-wiki** [comment]
Thank you!

**Pull request merged**